src/solvers/scip.rs

```diff
@@ -168,17 +307,15 @@ impl WithInitialSolution for SCIPProblem {
 /// A wrapper to a solved SCIP problem
 pub struct SCIPSolved {
     solved_problem: Model<Solved>,
-    id_for_var: HashMap<Variable, Rc<russcip::Variable>>,
+    id_for_var: HashMap<Variable, russcip::Variable>,
```

Comment:
The underlying russcip crate had breaking changes in https://github.com/scipopt/russcip/releases/tag/v0.6.0 when it removed the need for `Rc` on variables. good_lp could technically benefit from the same changes, but can we do these changes or is that considered a breaking change?

Reply:
Yes, it's a breaking change, but let's do it anyway. Increase the version number

Reply:
As in, updating the major version?

Reply:
yes

Reply:
I made the scip update and version bump separately, pushed to main.

Reply:
Thank you so much!